src/common/components/select-repository-list/index.js

```diff
@@ -196,9 +184,9 @@ export class SelectRepositoryListComponent extends Component {
               { selectedRepositories.length } selected
             </strong>
             {' '}
-            (<a href="#" onClick={this.onHelpClick.bind(this)}>
+            (<Button appearance={ 'link' } onClick={this.onHelpClick.bind(this)} aria-label={ this.state.showMissingReposInfo ? 'Click to close the missing / private repos dialog' : 'Click to open the missing / private repos dialog' }>
```

Comment:
@bartaz so with ```aria-labels``` you are to use them to explain the function in a more verbose way. This means it should describe the action it's trying to do or the page your trying to lead the user too. This is something we've done in MAAS and I feel this content would suffice.